### PR TITLE
fix(merge): update calling with latest master

### PR DESCRIPTION
**Pull request by sreenara**
# COMPLETES # NA

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

## The following scenarios where tested

UTs passed
Tested a guest joining a meeting with audio and video
Tested an outgoing call on the calling page

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-people/test/integration/spec/people.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('plugin-people', function () {
           assert.equal(person.emails[0], mccoy.email);
         }));
 
-      it('returns a list of people matching name', () =>
+      it.only('returns a list of people matching name', () =>
```
Comment (Neeraj-swarnkar) [inline packages/@webex/plugin-people/test/integration/spec/people.js:100]: @sreenara, why we are running only this test, should we remove only from this or this in intentional ?

**Comment by sreenara** [comment]
> Details

The min.js files came in because of the pull from master branch. Will remove them.

**Comment by sreenara** [comment]
The integration test failures can be ignored in this PR as these changes are not causing it.
Runs in https://github.com/webex/webex-js-sdk/pull/3034 have confirmed this

**Review by mkesavan13 — APPROVED** [review]
The changes don't seem to be very divergent. So, the better thing would have been to do a rebase so we carry the exact commits from the master. 
- That could help us Blame later in case of any questions on any code. This one will omit those commits
- When we try to merge calling into next again, we'll face conflicts because these changes might already be there in the `next` branch when @rajeshtezu/@rarajes2 synced them last sprint.

Otherwise looks good to me. Approving.

**Comment by sreenara** [comment]
This PR is no longer required. We are moving `calling` into `next` in #3040. 
Post this, the latest changes from `master` can be rebased into `next`.

**Pull request closed without merging**